#Description: Adjust manpages to talk abour editor instead of vi.
#Author: Michael Meskes <meskes@debian.org>

--- quota-3.16/edquota.8.orig	2004-02-12 15:33:11.000000000 +0100
+++ quota-3.16/edquota.8	2004-02-12 15:31:44.000000000 +0100
@@ -77,7 +77,7 @@
 the changes made.
 .LP
 The editor invoked is
-.BR vi (1)
+.BR editor (1)
 unless either the
 .SB EDITOR
 or the
@@ -166,7 +166,7 @@
 .PD
 .SH SEE ALSO
 .BR quota (1),
-.BR vi (1),
+.BR editor (1),
 .BR quotactl (2),
 .BR quotacheck (8),
 .BR quotaon (8),
--- quota/quotaops.c	2017-12-30 11:38:49.119348704 +0100
+++ quota/quotaops.c	2017-12-30 11:42:27.117257693 +0100
@@ -224,7 +224,7 @@
 			die(1, _("%s failed: %s\n"), "setuid", strerror(errno));
 		if (!(ed = getenv("VISUAL")))
 			if (!(ed = getenv("EDITOR")))
-				ed = _PATH_VI;
+				ed = "/usr/bin/editor";
 		i = 0;
 		ed = actp = sstrdup(ed);
 		while (actp) {
